crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -809,6 +811,7 @@ where
                         RethRpcModule::Txpool => {
                             TxPoolApi::new(self.pool.clone()).into_rpc().into()
                         }
+                        RethRpcModule::Rpc => RPCApi::new().into_rpc().into(),
```

Comment:
here we need to initialize the `RPCApi` with the selection of `RethRpcModule` over which we're looping, 
so ideally, we clone the selection before the loop and convert that into a hashmap<String,String> here